qlist.go

```diff
@@ -21,21 +21,34 @@ import (
 	"strings"
 )
 
-// qlist returns the results of executing ccontrol qlist for the specified instance.
+// qlist returns the results of executing qlist for the specified instance.
 // If instanceName is "", it will return the results of an argumentless qlist (which contains all instances)
 // It returns a string containing the combined standard input and output of the qlist command and any error which occurred.
 func qlist(instanceName string) (string, error) {
 	// Example qlist output...
 	// DOCKER^/ensemble/instances/docker/^2015.2.2.805.0.16216^down, last used Fri May 13 18:12:33 2016^cache.cpf^56772^57772^62972^^
+	// DOCKER^/ensemble/instances/docker^2018.1.1.643.0^running, since Mon Jul 23 14:42:09 2018^iris.cpf^1972^57772^62972^ok^IRIS^^^/ensemble/instances/docker
+	qlist := ""
 	args := []string{"qlist"}
 	if instanceName != "" {
 		args = append(args, instanceName)
 	}
 
-	out, err := exec.Command(globalCControlPath, args...).CombinedOutput()
-	if err != nil {
-		return "", err
+	if IrisExists() {
```

Comment:
I would extract and centralize this logic but I need clarification on the `*Exists` methods to say exactly how.